src/system_sensors.py

```diff
@@ -197,6 +222,8 @@ def on_message(client, userdata, message):
 
     devicename = settings['devicename'].replace(' ', '').lower()
     deviceNameDisplay = settings['devicename']
+    deviceModel = get_host_model()
+    deviceManufacturer = "RPI Foundation" if "rasp" in OS_DATA["ID"] else OS_DATA['Name']
```

Comment:
Should `'Name'` be `’NAME’`

It’s uppercase on my RPi and x86 boxes.

Reply:
Nice catch 👍